apollo/integrations/db/salesforce_crm_proxy_client.py

```diff
@@ -39,6 +39,43 @@ def execute(self, query: str) -> Dict:
 
         return {"description": description, "records": records, "rowcount": rowcount}
 
+    def execute_count_query(self, query: str) -> Dict:
+        headers = {
+            "Sforce-Query-Options": "batchSize=200",
+        }
+        results = self._connection.query(query, headers=headers)
```

Comment:
I'm going to have to rework both `execute()` and `execute_row_limit()` to use `query_all_iter()` instead of `query()` so I can iterate through all results when there are a lot of rows returned. 

(For `execute_count_query()` I get the totalSize in the first batch and throw away the result rows, so I can get away with using `query()` and getting a single batch of results.)